tree/dataframe/inc/ROOT/RDFActionHelpers.hxx

```diff
@@ -323,6 +324,69 @@ public:
    HIST &PartialUpdate(unsigned int slot) { return *fTo->GetAtSlotRaw(slot); }
 };
 
+class FillTGraphHelper : public ROOT::Detail::RDF::RActionImpl<FillTGraphHelper> {
+public:
+   using TGraph = ::TGraph;
```

Comment:
I would if possible use another name, for example `using TGraphClass = ::TGraph` to avoid messes with the names.

Reply:
By the way, I am confused. Why do we need this renaming in the first place :-) ?